org-formation/710-tgw/vpc.yaml

```diff
@@ -0,0 +1,203 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >-
+  VPC for a Transit Gateway
+Parameters:
+  VpcSubnetPrefix:
+    Description: The VPC subnet prefix (i.e. 10.40)
+    Type: String
+Mappings:
+  SubnetConfig:
+    VPC:
+      CIDR: "0.0/16"
+    SubnetA:  # private
+      CIDR: "1.0/24"
```

Comment:
If 24 bits are specified, shouldn't there be 3 digits in the IPv4 address, like "1.0.0/24"?